tools/harness-thci/OTBR.py

```diff
@@ -0,0 +1,2655 @@
+    def setNetworkDataRequirement(self, eDataRequirement):
+        """set whether the Thread device requires the full network data
+           or only requires the stable network data
+
+        Args:
+            eDataRequirement: is true if requiring the full network data
+
+        Returns:
+            True: successful to set the network requirement
+        """
+        print '%s call setNetworkDataRequirement' % self.port
+        print eDataRequirement
+
+        if eDataRequirement == Device_Data_Requirement.ALL_DATA:
+            self.networkDataRequirement = 'n'
+        return True
+
+    def configExternalRouter(self, P_Prefix, P_stable, R_Preference=0):
+        """configure border router with a given external route prefix entry
+
+        Args:
+            P_Prefix: IPv6 prefix for the route
+            P_Stable: is true if the external route prefix is stable network data
+            R_Preference: a two-bit signed integer indicating Router preference
+                          1: high
+                          0: medium
+                         -1: low
+
+        Returns:
+            True: successful to configure the border router with a given external route prefix
+            False: fail to configure the border router with a given external route prefix
+        """
+        print '%s call configExternalRouter' % self.port
+        print P_Prefix
+        stable = ''
+        prefix = self.__convertIp6PrefixStringToIp6Address(str(P_Prefix))
+        try:
+            if P_stable:
+                cmd = self.otbrCmd + 'add-route %s -l 64 -p %d' % (prefix, R_Preference)
+            else:
+                cmd = self.otbrCmd + 'add-route %s -l 64 -p %d -n' % (prefix, R_Preference)
+            print cmd
+
+            if self.__sendCommand(cmd)[0] != 'Fail':
+                return True
+            else:
+                return False
+        except Exception, e:
+            ModuleHelper.WriteIntoDebugLogger("configExternalRouter() Error: " + str(e))
+
+    def getNeighbouringRouters(self):
+        """get neighboring routers information
+
+        Returns:
+            neighboring routers' extended address
+        """
+        print '%s call getNeighbouringRouters' % self.port
+        try:
+            routerInfo = []
+            routerList = []
+            routerList = self.__sendCommand(self.otbrCmd + 'getprop -v Thread:NeighborTable')
+            print routerList
+
+            if 'Done' in routerList:
+                print 'no neighbouring routers'
+                return None
+
+            for index in routerList:
+                router = []
+                cmd = 'router %s' % index
+                router = self.__sendCommand(cmd)
+
+                for line in router:
+                    if 'Done' in line:
+                        break
+                    #elif 'Rloc' in line:
+                    #    rloc16 = line.split()[1]
+                    elif 'Ext Addr' in line:
+                        eui = l
```

Comment:
Maybe the command should be commissioner --start??
wpanctl:wpan0> commissioner start
wpanctl:wpan0> commissioner --start
Commissioner command applied.